src/shared/theme/themes.ts

```diff
@@ -83,6 +84,7 @@ export const baseTheme: Theme = {
     textMuted: 'rgb(150, 150, 150)',
     accent: 'rgb(255, 67, 62)',
     accentTransparent: 'rgba(255, 67, 62, 0.75)',
+    accentMuted: 'rgba(255, 67, 62, 0.4)',
```

Comment:
`accentTransparent` was a little too jarring, so I applied the same `0.4` opacity as I did with the `accentTwoMuted` 

https://github.com/Kava-Labs/oros/pull/388/files#r1968356748

Reply:
I still think the orange warning may. be a bit jarring. Might be worth swapping in the purple and adding a screenshot to get feedback from Kevin.